docs-developer/references/release_process.rst

```diff
@@ -162,23 +176,71 @@ Release with PyPI using the make command::
 
     $ make release
 
-Declare victory.
 
-Post-release TODO
```

Comment:
Isn't this somehow missing? Such that the reader will know that the new steps defined are supposed to be done *after* a release?

Reply:
Maybe it depends on what you define "releasing" as?

Here, I was imagining that _all_ steps listed are necessary to "completing a release", and the steps are generally meant to be followed in order.

Do you have a different mental model of what constitutes "releasing", and what constitutes "post-release"?

Reply:
Well, yes, maybe it's my definition -- at least I have to admit that it's out of personal experience, that I'd want the wording to guide the intuition correctly.

Reply:
I'd like to use terms as intuitively as possible!

Let me know if you have any recommendations; otherwise mind approving?

we can keep iterating on this as necessary

Reply:
I agree! :)